pkg/services/publicdashboards/service/service.go

```diff
@@ -459,37 +459,6 @@ func (pd *PublicDashboardServiceImpl) Delete(ctx context.Context, uid string, da
 	return pd.serviceWrapper.Delete(ctx, uid)
 }
 
-func (pd *PublicDashboardServiceImpl) DeleteByDashboard(ctx context.Context, dashboard *dashboards.Dashboard) error {
```

Comment:
I moved all deletions to the public dashboard service wrapper to avoid dependency issues.
Now, the dashboard service is responsible for calling the public dashboard deletion.

Reply:
I agree with the decision but I think it breaks enterprise, since the enterprise service wrapper should implement this new `DeleteByDashboardUIDs` method that will just call the oss wrapper service. Am I correct?